src/components/pipeline-list/pipeline-list.scss

```diff
@@ -14,21 +14,12 @@
   --pipeline-list-shadow: rgba(black, 0.1);
   --pipeline-list-label-bg: #{$color-bg-dark-5};
   --pipeline-list-options-bg: #{$color-bg-dark-1};
-  --pipeline-list-option-border-width: 0 0 1px;
+  --pipeline-list-option-border-width: 0 0 1px 2px;
```

Comment:
This new border setting doesn't seem to be related with the relevant changes with kedro-UI - is this a small visual enhancement that you noticed during development?